e2etests/testdata/cel-config.yaml

```diff
@@ -0,0 +1,23 @@
+checks:
+  addAllBuiltIn: false
+customChecks:
+  - name: "cel-forbidden-annotation"
+    description: "cel espression cehck"
+    remediation: "Remove reloader.stakater.com/auto annotation"
+    scope:
+      objectKinds:
+        - DeploymentLike
+    template: "cel-expression"
+    params:
+      check: |
+        has(subject.metadata.annotations) && subject.metadata.annotations["reloader.stakater.com/auto"] == "true" ? "Object has reloader annotation" : ""
```

Comment:
So, do we call the _thing_ subject or object? And if we're confused, maybe `target`?